opentimelineio/adapters/cmx_3600.py

```diff
@@ -721,7 +729,7 @@ def read_from_string(input_str, rate=24, ignore_timecode_mismatch=False):
     return result
 
 
-def write_to_string(input_otio, rate=None, style='avid'):
+def write_to_string(input_otio, rate=None, style='avid', trunc_reelname=True):
```

Comment:
Instead of passing in `trunc_reelname` you could pass in `reelname_len` (by default 8), which if you pass in `None` would not truncate at all?  Or is it _always_ 8?

Reply:
Good idea. The standard is 8, but a lot of apps use longer reelnames. This is a better way of supporting that. 
Do you have any thought on the compare test of the two jsons that I mentioned? I can always create/modify the source json to what i expect and compare that?

Reply:
Could you use the optional `flavor` argument to include the new stuff, keeping the old stuff as-is?
Also, it looks like the tests are failing in Travis due to some linter issues. Once that goes further to the EDL tests, then it will be easier for us to see the diffs you are talking about.

Reply:
When you say "the new stuff", do you mean the new comment and metadata or do you mean everything? The 8 character reelname length I believe is in the cmx3600 specs and should perhaps be part of that flavor? As for the new comment line that was just an idea to inform users about the truncated reelname as discussed in issue #168.
I'll fix the lint errors asap. Thought I caught all of them. Sorry about that.

Reply:
@jminor lint fixes are done. It now fails on the compare test mentioned above. 
@ssteinbach I'm working on applying your comments, but have one question. When passing `None` to reelname_len, should I still strip the extension of the source file in the reelname?

Reply:
@apetrynet Those diffs in the test just look like you need to update the baselines.  You're adding more metadata, correct?

Reply:
@ssteinbach Yes, I add the reelname to metadata when parsing an EDL.
Should I just add the expected metadata to the clips in `test_edl_round_trip_mem2disk2mem`?